src/classrooms/classrooms.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Post, UseGuards, Req } from "@nestjs/common";
+
+import { AuthenticatedRequest } from "@/auth/auth.interfaces";
+import { JwtAuthGuard } from "@/auth/guards/jwt-auth.guard";
+
+import { ClassroomResponseDto, CreateClassroomDto } from "./classrooms.dtos";
+import { ClassroomsService } from "./classrooms.service";
+
+@Controller("classrooms")
+@UseGuards(JwtAuthGuard)
+export class ClassroomsController {
+  constructor(private readonly classroomsService: ClassroomsService) {}
+
+  @Post()
+  createClassroom(
+    @Req() req: AuthenticatedRequest,
```

Comment:
If you want the current user id, you can use the @CurrentUser decorator. You can find it in the template